lib/glific/partners/out_of_office.ex

```diff
@@ -14,15 +14,17 @@ defmodule Glific.Partners.OrganizationSettings.OutOfOffice do
     :enabled,
     :start_time,
     :end_time,
-    :flow_id
+    :flow_id,
+    :default_flow_id
```

Comment:
We might need to change this struct name to default_flows instead of out_of_office.ex